modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/AbstractChangeIndexStatusEntry.java

```diff
@@ -43,7 +43,7 @@ abstract class AbstractChangeIndexStatusEntry implements UpdateEntry {
     }
 
     @Override
-    public Catalog applyUpdate(Catalog catalog, long causalityToken) {
+    public final Catalog applyUpdate(Catalog catalog, long causalityToken) {
```

Comment:
To make it explicit that `applyUpdate` is defined on this class and should not be overriden in the subclasses as this class is designed to implement `applyUpdate` itself (as it seems).

Reply:
I think this is unnecessary, we don’t seem to practice this, at your discretion.

Reply:
I would prefer to leave it as is: it doesn't create any real problems, but serves as a reminder to someone who accidentally tries to override this method. If it's really needed, it will be easy just to remove this `final`.